components/productions/index/search.vue

```diff
@@ -29,5 +29,6 @@
 <style lang="scss" scoped>
   .searchbox {
     margin-bottom: 1rem;
+    max-width: 43rem;
```

Comment:
Thanks for moving this, but it's still used in multiple places. Maybe we should move it to a variable to prevent a wide range of max-widths in the long run. That way the number can also get a descriptive name to make it seem less arbitrary. (I'm still not sure why `43rem` exactly?)

Reply:
This is also a relevant discussion for the margins, maybe we should have global variables like `width sm, md, lg` and `margin sm, md, lg` in our global.scss?

Reply:
It can be more values and differently named